universal/CMakeLists.txt

```diff
@@ -70,7 +75,7 @@ function (add_universal_target TARGET SOURCE)
                 "-DUNIVERSAL_PROJECT_DIR:FILEPATH=${SOURCE}"
                 "-DUNIVERSAL_BINARY_DIR:FILEPATH=${CMAKE_CURRENT_BINARY_DIR}/${TARGET}/${platform}"
                 "-DSOURCE_TARGET=${SOURCE_TARGET}"
-                "-Dpicotool_DIR=${picotool_INSTALL_DIR}/picotool"
+                "-Dpicotool_DIR=${universal_picotool_DIR}"
                 "-Dpioasm_DIR=${PIOASM_INSTALL_DIR}/pioasm"
```

Comment:
Is a similar "fix" also needed here for `pioasm_DIR` ? (I've not experimented with a standalone-pioasm yet)

Reply:
I've pushed a commit to your branch to make a similar change for `pioasm_DIR`, I hope that was okay @will-v-pi !

Reply:
Yep, that's great - thanks